pyxnat/tests/resources_test.py

```diff
@@ -80,63 +81,64 @@ def test_multi_create():
     assert asse_2.datatype() == 'xnat:petAssessorData'
     assert expe_2.datatype() == 'xnat:petSessionData'
 
-    scan_2 = central.select('/projects/nosetests/subjects/%(sid)s'
+    scan_2 = central.select('/projects/nosetests3/subjects/%(sid)s'
                             '/experiments/%(eid)s/scans/%(cid)s' % _id_set2
                             ).create()
 
     assert scan_2.datatype() == 'xnat:mrScanData'
 
 #def test_share_subject():
-#    assert not central.select('/projects/nosetests2'
+#    assert not central.select('/projects/nosetests32'
```

Comment:
Is just a commented fragment but watch out since `nosetests32` project does not exist

Reply:
Fixed in 407bce4.